src/adapters/curve/tests/snapshots/ethereum.profits.profits-2.json

```diff
@@ -32,43 +32,13 @@
           "symbol": "3Crv",
           "decimals": 18,
           "type": "protocol",
-          "profit": 0,
-          "performance": 0,
+          "profit": 3.15529238292811e-8,
+          "performance": 4.90916972549114e-9,
```

Comment:
I'm going to assume this is somehow intended, but just want to double check.

Before the reward adapter was returning independently (and without the `fromBlock` and `toBlock`) and now it is part of the profits response.